frame/vamm/src/lib.rs

```diff
@@ -341,7 +411,121 @@ pub mod pallet {
 			})
 		}
 
-		fn get_twap(vamm_id: &Self::VammId) -> Result<Self::Decimal, DispatchError> {
+		/// Gets the current price of the __base__ or __quote__ asset in a vamm.
+		///
+		/// # Overview
+		/// In order for the caller knows which is the current price of an asset
+		/// in an specific vamm it has to request it to the Vamm Pallet. The
```

Comment:
'In order for the caller to know what the current price of an asset in a specific vamm is, it has to request it to the Vamm Pallet'?

Reply:
Definitely better, thanks!